src/modules/31-variables/strings/strings.en.yaml

```
@@ -4,6 +4,12 @@ allowedValues: Allowed Values
 noVariableFound: No Variable Found
 noVariableExist: There are no variables in your {{resourceName}}.
 inputValidation: Input Validation
+confirmDeleteTitle: Delete Variable
+editVariable: Edit the Variable
+confirmDelete: Are you sure you want to delete the Variable '{{name}}'
```

Probably should have a `?`
```suggestion
confirmDelete: Are you sure you want to delete the Variable '{{name}}'?
```